pkg/module/disk_ref_test.go

```diff
@@ -0,0 +1,59 @@
+package module
+
+import (
+	"io/ioutil"
+	"path/filepath"
+)
+
+func (m *ModuleSuite) TestDiskRefReadAndClear() {
+	const (
+		root    = "testroot"
+		version = "v1.0.0"
+		info    = "testinfo"
+		mod     = "testmod"
+		zip     = "testzip"
+	)
+	r := m.Require()
+
+	// create a new disk ref using the filesystem
+	diskRef := newDiskRef(m.fs, root, version)
+
+	// ensure that reading fails, because there are no files
+	ver, err := diskRef.Read()
+	r.Nil(ver)
+	r.NotNil(err)
+
+	// create all the files the disk ref expects
+	infoFile, err := m.fs.Create(filepath.Join(root, version+".info"))
+	r.NoError(err)
+	defer infoFile.Close()
```

Comment:
Is it okay to call close twice? because the `diskref.Clear()` will do it as well.